otel-tests/src/lib.rs

```diff
@@ -0,0 +1,65 @@
+use opentelemetry::sdk::trace;
+use opentelemetry::sdk::Resource;
+use opentelemetry::KeyValue;
+use opentelemetry_otlp::WithExportConfig;
+use std::sync::Once;
+use tracing_subscriber::filter::LevelFilter;
+use tracing_subscriber::layer::SubscriberExt;
+use tracing_subscriber::util::SubscriberInitExt;
+use tracing_subscriber::EnvFilter;
+use tracing_subscriber::Layer;
+use tracing_subscriber::Registry;
+
+pub use futures;
+pub use opentelemetry;
+pub use otel_tests_macro::otel_test;
+pub use tokio;
+
+static INIT_OTLP_EXPORTER: Once = Once::new();
+
+pub fn init_tracing() {
+    INIT_OTLP_EXPORTER.call_once(|| {
+        let env = std::env::var("ITCHYSATS_TEST_INSTRUMENTATION").unwrap_or_default();
+        let telemetry = if env == "1" {
+            let cfg = trace::Config::default().with_resource(Resource::new([KeyValue::new(
+                "service.name",
+                "daemon-tests",
+            )]));
+
+            let tracer = opentelemetry_otlp::new_pipeline()
+                .tracing()
+                .with_trace_config(cfg)
+                .with_exporter(
+                    opentelemetry_otlp::new_exporter()
+                        .grpcio()
+                        .with_endpoint("localhost:4317"),
```

Comment:
❓ Should/can this be configurable? Or is it okay to just hard-code it?

Reply:
This is okay to hard code, I think, given it will be run locally with jaeger.sh